include/aos/sm/launcher.hpp

```diff
@@ -30,6 +30,14 @@ namespace launcher {
  *  @{
  */
 
+/**
+ * Launcher operation version.
+ * If new functionality doesn't allow existing services to work
+ * properly, this value should be increased. It will force to remove all
+ * services and their storages before first start.
+ */
+constexpr auto cOperationVersion = AOS_CONFIG_LAUNCHER_OPERATION_VERSION;
```

Comment:
It is pure const and isn't supposed to be changed with config. Make it either public const of Launcher or private const and add GetOperationVersion method to launcher itf.